python/robyn/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: robyn.tech
+  github: sparckles/robyn
+  version: 0.54.0
+
+  engines:
+    - robyn
+
+language:
+  engines:
+    robyn:
+      command: python /usr/src/app/server.py --workers=$(nproc) --processes=$(nproc) --log-level=WARN
```

Comment:
I think WARN coul mess up the result @sansyrox @jackcvr. Indeed the purpose here is to reflect a production usage, but IO operations could mess up results.

That's why logging is disabled (or should be) on all frameworks here

Reply:
The easiest way to reduce logs could be to set level to CRITICAL. Will it fit?